src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
```

Comment:
The `class` attribute should be on the same line as the opening `<html>` tag if it fits within the 80 character limit.